tests/test_vpc.rb

```diff
@@ -34,20 +35,12 @@ def no_feature_vpc
     config('terminal dont-ask ; no feature vpc')
```

Comment:
Nit: This is okay but I'd like to see us avoid using config() when possible. In this case you can achieve the same result with something like this:

```
domain = Vpc.domains
domain.destroy unless domain.empty?
```